### ISSUE TITLE
Center Mode - centerPadding default being used

### ISSUE THREAD

**Issue by wgstjf** [kenwheeler/slick#1078]
Hi there,

I have a problem with a CenterMode slider I have built.  When the slider initially loads (Ctrl + Shift + R in chrome) the slider correctly calculates the width of the slides (which have the image set as a background for the slide).  When the page loads again (Refresh in Chrome with Ctrl + R) the slider incorrectly caclulates the width of the list by instead of using the CenterPadding variable that I initialise with it uses your default of 50px.  This makes the slides not align correctly.  

In my example the centerPadding variable is calculated by getting the width of an element that is known to be the full width of the content area of the site, then subtracting that from the window width and dividing by 2 (the appending 'px').  It is then passed to slick in the init for the slider, eg: centerPadding: centerPaddingCalculated,  

In short slick.min.js (v.1.4.1) seems to sometimes take the variable passed to it and other times use the default. 

Any help would be greatly appreciated!

Cheers,

Will

**Comment by desandro** [comment]
You might want to try [Flickity](http://flickity.metafizzy.co). Flickity allows you to use your own CSS to handle size and positioning of the cells in your slider, so you may not run into this issue.

**Comment by wgstjf** [comment]
@desandro thanks for the suggestion, as per my other comment a wonderful suggestion that has achieved exactly what I needed.

Cheers,

Will
